judge/migrations/0132_site_data_batch_prerequisites.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.27 on 2022-03-07 19:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0131_spectate_contests'),
```

Comment:
This is so out of date it makes me sad :(

Reply:
Rebased. This is old, wow.